alteryx/woodwork
# v0.0.11

**v0.0.11 March 15, 2021**
- Changes
  - Restrict Koalas version to ``<1.7.0`` due to breaking changes (#674)
  - Include unique columns in mutual information calculations (#687)
  - Add parameter to include index column in mutual information calculations (#692)
- Documentation Changes
  - Update to remove warning message from statistical insights guide (#690)
- Testing Changes
  - Update branch reference in tests to run on main (#641)
  - Make release notes updated check separate from unit tests (#642)
  - Update release branch naming instructions (#644)
